torch_xla/csrc/init_python_bindings.cpp

```diff
@@ -1922,22 +1922,22 @@ void InitXlaModuleBindings(py::module m) {
   distributed_runtime_service.def("shutdown",
                                   &xla::DistributedRuntimeService::Shutdown,
                                   py::call_guard<py::gil_scoped_release>());
-  m.def("_xla_get_distributed_runtime_service",
-        [](int num_nodes) -> std::unique_ptr<xla::DistributedRuntimeService> {
-          std::string dist_service_addr =
-              runtime::sys_util::GetEnvString("PJRT_DIST_SERVICE_ADDR", "");
-          XLA_CHECK(!dist_service_addr.empty())
-              << "Must set PJRT_DIST_SERVICE_ADDR environment variable to use "
-                 "distributed runtime";
-          XLA_CHECK(num_nodes > 0)
-              << "num_nodes must be positive: " << num_nodes;
-
-          xla::CoordinationServiceImpl::Options options;
-          options.num_nodes = num_nodes;
-          return std::move(
-              xla::GetDistributedRuntimeService(dist_service_addr, options)
-                  .value());
-        });
+  m.def(
+      "_xla_get_distributed_runtime_service",
+      [](int num_nodes) -> std::unique_ptr<xla::DistributedRuntimeService> {
+        std::string master_addr =
+            runtime::sys_util::GetEnvString("MASTER_ADDR", "localhost");
+        std::string port =
+            runtime::sys_util::GetEnvString("XLA_COORDINATOR_PORT", "8547");
```

Comment:
I'm concerned about hard-coding the default value in two places, can we make this a const somewhere? Or if you want, this can be addressed in the follow-up to move everything into C++.

Reply:
Fair. Addressed it in https://github.com/pytorch/xla/pull/5714